packages/rebber/src/types/linkReference.js

```diff
@@ -1,6 +1,6 @@
 module.exports = linkReference
 
-const defaultMacro = (reference, inner) => `${inner}\\ref{${reference}}`
+const defaultMacro = (reference, inner) => `\\href{${reference}}{${inner}}`
```

Comment:
These two are not the same in LaTeX, as pointed below

Reply:
yep and `\ref` was not really the good command imho.